src/porepy/numerics/fem/rt0.py

```diff
@@ -331,6 +313,62 @@ def extract_p(self, g, up):
 
     # ------------------------------------------------------------------------------#
 
+    def project_u(self, g, u, data):
```

Comment:
This should be mentioned in a release note for this update

Reply:
It does, it's actually a great idea!